config/default.yml

```diff
@@ -3432,6 +3432,12 @@ Style/EvenOdd:
   VersionAdded: '0.12'
   VersionChanged: '0.29'
 
+Style/ExcessiveNumberedParameters:
```

Comment:
I think we can probably drop the `Excessive` prefix, as (as far as I know) all cops are named objectively by what they check, e.g. `LineLength` and not `ExcessiveLineLength`. Perhaps (if needed) replaced by a suffix like in `NumberedParameterCount`. WDYT?

Reply:
`NumberedParameterCount` works for me!

Reply:
Or NumberedParamatersLimit. Not sure what names we used in the past in similar situations.

Reply:
I have renamed the cop to `Style/NumberedParametersLimit`.